logzio-sender/src/main/java/io/logz/sender/LogzioSender.java

```diff
@@ -265,7 +245,7 @@ private int sizeInBytes(List<FormattedLogMessage> logMessages) {
 
     private byte[] toNewLineSeparatedByteArray(List<FormattedLogMessage> messages) {
         try (ByteArrayOutputStream byteOutputStream = new ByteArrayOutputStream(sizeInBytes(messages));
-             OutputStream os = compressRequests ? new GZIPOutputStream(byteOutputStream) : byteOutputStream) {
+             OutputStream os = httpsRequestConfiguration.isCompressRequests() ? new GZIPOutputStream(byteOutputStream) : byteOutputStream) {
```

Comment:
This should be inside the request sender I believe.

Reply:
@asafm 
I feel like this should be in the java sender and not at the sync client, I want the client to be as simple as possible - getting msg and configuration and that's it. Whoever uses it should make sure the msg is byte[]. What do you think?